dts/arm/st/h7/stm32h723XX.dtsi

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/h7/stm32h723.dtsi>
+
+/ {
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
```

Comment:
Why not putting this in stm32h723.dtsi ? This will be common to all the soc package variants.

Reply:
Done